snippets/ponylang-mode/main.block

```diff
@@ -3,8 +3,10 @@
 # name: block main
 # contributor: Sean T. Allen <sean@monkeysnatchbanana.com>
 # --
-actor Main =>
+actor Main
```

Comment:
This is a pretty good change, can you do it in a new PR without the hello world being printed?

Reply:
yes